### PR TITLE
Learning Objectives for Lessons 12 and 13.

### PR DESCRIPTION
**Pull request by gildedgardenia**
Put together the learning objectives for the HTML, CSS, and Git lessons based off of the old learning objectives.

### CODE REVIEW

File: unit1/lesson12.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Objectives
+----------
+
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
```
Comment (jimflores5) [inline unit1/lesson12.rst:20]: We should also include the ``<hr>`` tag, either here or with the "additional tags".  Also, if we are going to introduce the bold and italic tags, we should include underline as well.

File: unit1/lesson12.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
```
Comment (jimflores5) [inline unit1/lesson12.rst:10]: Would it be appropriate to address semantic html here or in a future lesson?

File: unit1/lesson12.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
+- Define HTML attributes and how they are used
+- Describe how whitespace in HTML documents effects rendering
+- Describe inline and block display; for common elements identify their default display
+- Describe HTML document structure and associated elements: doctype, <html>, <head>, <body>, <title>
+- Write a properly-structured document head.
+- Describe the purpose of, and use additional HTML tags: <ul>, <ol>, <li>, <table>
```
Comment (jimflores5) [inline unit1/lesson12.rst]: Grammar: "Describe the purpose and use of..."

File: unit1/lesson12.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- Describe the main functionality provided by a version control system (VCS)
+- Describe the practical applications of this functionality: revision history, collaboration, backup
+- Explain what a repository is
+- Explain what a commit is
+- Describe the basic commit flow and commands: status, add, commit, log
+- Initialize a new Git repository
+- Commit to a local repository
+- Clone a project from a remote Git repository
+- Explain the two Git file status: tracked and untracked
+- Explain what types of changes should be within a commit
```
Comment (jimflores5) [inline unit1/lesson12.rst:40]: Will git push/pull be addressed here or in the next lesson?

File: unit1/lesson13.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- Explain how CSS allows for separation of display and document structure
+- Describe the 3 locations that style rules may be placed: external, document, inline
+- Use CSS rules at each of the 3 possible locations
+- Describe precedence between CSS rules based on location
+- Use basic CSS rule types: font-size, color, font-family, background-color, text-align, etc
+- Describe the following simple selector types, and situations in which each is preferred: element, ID, class
+
+Git
+^^^
+
```
Comment (jimflores5) [inline unit1/lesson13.rst:27]: From my own experience going through LC101 and Liftoff, I think it would be beneficial to provide a more detailed explanation about the differences between 'git fetch' and 'git pull'.  When is it appropriate to use one over the other?

File: unit1/lesson12.rst [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+^^^
+
+- Describe the main functionality provided by a version control system (VCS)
+- Describe the practical applications of this functionality: revision history, collaboration, backup
+- Explain what a repository is
+- Explain what a commit is
+- Describe the basic commit flow and commands: status, add, commit, log
+- Initialize a new Git repository
+- Commit to a local repository
+- Clone a project from a remote Git repository
```
Comment (jimflores5) [inline unit1/lesson12.rst:38]: Students should also know how to accomplish the reverse of this - how to create a remote repo (or branch) from a local repo.  (Not sure if this topic belongs here or in the next lesson).

File: unit1/lesson12.rst (hunk 2 above)
Comment (chrisbaylewis) [reply unit1/lesson12.rst:10]: I'm 100% on board with teaching semantic HTML, but it feels like a topic that requires more nuance than we have time for in 1-2 classes that also introduce a lot of other topics. My instinct is that we can briefly mention the topic and provide a link to something like [HTML 5 Doctor](http://html5doctor.com/element-index/). If we build a front-end focused unit 2, that would be the place to cover semantic HTML properly, and in-full.

File: unit1/lesson12.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
+- Define HTML attributes and how they are used
+- Describe how whitespace in HTML documents effects rendering
+- Describe inline and block display; for common elements identify their default display
```
Comment (chrisbaylewis) [inline unit1/lesson12.rst]: This feels like it would be better placed in the CSS section.

File: unit1/lesson12.rst [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+- Describe the main functionality provided by a version control system (VCS)
+- Describe the practical applications of this functionality: revision history, collaboration, backup
+- Explain what a repository is
+- Explain what a commit is
+- Describe the basic commit flow and commands: status, add, commit, log
+- Initialize a new Git repository
+- Commit to a local repository
+- Clone a project from a remote Git repository
+- Explain the two Git file status: tracked and untracked
```
Comment (chrisbaylewis) [inline unit1/lesson12.rst]: "status" should be plural, "statuses"

File: unit1/lesson13.rst [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+- Learn how Git can be used to manage teamwork.
+
+Objectives
+----------
+
+CSS
+^^^
+
+- Explain how CSS allows for separation of display and document structure
+- Describe the 3 locations that style rules may be placed: external, document, inline
```
Comment (chrisbaylewis) [inline unit1/lesson13.rst]: As an addendum to this objective, or a new objective itself, it seems like students should understand when one location may be preferred over another for style rules.

File: unit1/lesson12.rst (hunk 6 above)
Comment (chrisbaylewis) [reply unit1/lesson12.rst:38]: It seems that this is included in the last objective for class 13: "Create and push to a remote repository." The only possible addition to that one to make it fully clear could be something about how to connect a remote repo to a local repo, which is implicit in this objective but could be explicitly stated.

File: unit1/lesson13.rst [hunk 10]
```diff
@@ -0,0 +1,36 @@
+Unit 1 - Lesson 2 Learning Objectives
```
Comment (chrisbaylewis) [inline unit1/lesson13.rst]: Typo: "Lesson 2" -> "Lesson 13"

**Review by chrisbay — CHANGES_REQUESTED** [review]
I added a few minor suggestions, but this looks great overall.

File: unit1/lesson12.rst (hunk 2 above)
Comment (jimflores5) [reply unit1/lesson12.rst:10]: What if we simply begin the HTML training with semantic concepts?  Rather than train with the standard tags, could we begin with ``<section>``, etc. instead of ``<div>``?

File: unit1/lesson12.rst (hunk 7 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst]: Ok! What about moving whitespace and inline and block display to CSS? I think tags and attributes go naturally together and it is easier to learn them as part of HTML.

File: unit1/lesson12.rst (hunk 8 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst]: Good catch!

File: unit1/lesson13.rst (hunk 9 above)
Comment (gildedgardenia) [reply unit1/lesson13.rst]: I like that as an addendum. I think it will flow nicely!

File: unit1/lesson13.rst (hunk 10 above)
Comment (gildedgardenia) [reply unit1/lesson13.rst]: Thanks!

File: unit1/lesson12.rst (hunk 2 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst:10]: I think semantic HTML is the way to go. As a beginner, I remember finding it easier to visualize what the end result of my code with semantic HTML. Perhaps the differences between semantic HTML and HTML would actually be great booster rocket topic!

File: unit1/lesson12.rst (hunk 3 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst]: That would actually change the meaning of the sentence. I will reword it to clarify the sentence better. 

"Describe the purpose of and be able to use additional HTML tags..."

File: unit1/lesson12.rst (hunk 4 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst:40]: Git push will be part of lesson 12, but pull won't be used until lesson 13.

File: unit1/lesson13.rst (hunk 5 above)
Comment (gildedgardenia) [reply unit1/lesson13.rst:27]: I think that is more of a Liftoff topic. The best way to understand the difference between git fetch and git pull is in the context of a project. Students won't use Git very collaboratively until Liftoff.

File: unit1/lesson12.rst (hunk 6 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst:38]: I can reword that objective in Lesson 13 to make it clearer that that is what it means.

File: unit1/lesson12.rst (hunk 1 above)
Comment (gildedgardenia) [reply unit1/lesson12.rst:20]: I think in HTML 5, the ``<hr>`` tag became a thematic break between multiple topics. I do know that some people don't particularly like the underline because it can be confused very easily for a hyperlink.

File: unit1/lesson12.rst (hunk 7 above)
Comment (chrisbaylewis) [reply unit1/lesson12.rst]: My comment was only in reference to the inline vs. block stuff. The rest seems to make sense when covering HTML.

**Review by chrisbay — APPROVED** [review]
Ready to merge!
